spec/refresher_spec.rb

```diff
@@ -5,78 +5,78 @@
   context "guess the collection" do
 
     it "Yes, it is a Object, but which class" do
-      expect([].is_a? _fill_in_object_).to be true
-      expect(Array.new.instance_of? _FILL_ME_IN_).to be true
+      expect([].is_a? Object).to be true
```

Comment:
Technically true :) - I think we wanted to just show that both the literal `[]` and the actual class `Array` can be used for creating and managing lists.